lib/pages/add_wallet_views/restore_wallet_view/restore_options_view/restore_options_view.dart

```diff
@@ -437,6 +441,32 @@ class _RestoreOptionsViewState extends ConsumerState<RestoreOptionsView> {
                     color: Colors.transparent,
                     child: Column(
                       children: [
+                        Row(
+                          children: [
+                            Checkbox(
```

Comment:
We have https://github.com/cypherstack/stack_wallet/blob/staging/lib/widgets/custom_buttons/checkbox_text_button.dart which might be easier to use here

Reply:
Would probably be more consistent, UI-wise, anyways.  One min